ui/src/components/Section/section.scss

```diff
@@ -85,6 +85,9 @@
   .section__inner {
     /* Size intentions */
     &.section--padding {
+    &.section--notop {
```

Comment:
to keep this uniform with our current naming conventions lets rename it to section--padding__top-0

Reply:
Lets also make a bottom version of this in case we need to remove padding from top and/or bottom